zen/server.c

```diff
@@ -149,6 +159,9 @@ zn_server_create(struct wl_display *display)
   char socket_name_candidate[16];
   char *xdg;
 
+  zn_assert(
+      server_singleton == NULL, "Tried to create zn_server multiple times");
```

Comment:
```suggestion
      server_singleton != NULL, "Tried to create zn_server multiple times");
```
I could be wrong, but isn't it like this?

Reply:
`zn_assert` is check first argument, and if it is **false**, show message and return false.
In this case, it expects this is the first time that creating zn_server, so `server_singleton` must be NULL. So, it looks correct.

Reply:
that's right...
I misunderstood. Thx!